extensions/jitsi/common/client.mjs

```diff
@@ -125,7 +125,12 @@ class Client {
   /**
    * Start the Jitsi Meeeting
    */
-  startMeeting(meetingName, password) {
+  async startMeeting(meetingName, password) {
+    // if this.jitsiObj is set, the previous meeting has not yet ended.
+    while (this.jitsiObj) {
```

Comment:
optional: Consider awaiting a Promise that is resolved when this.jitsiObj is destroyed. Note that this path can be a little bit convoluted and the benefits may not be so apparent so consider making this a TODO.